### ISSUE TITLE
An empty directory is still created in Downloads after change the default location

### ISSUE THREAD

**Issue by rpxomi** [frostwire/frostwire#821]
Hi, by changing the default settings of the directory where the downloads will be saved, an empty folder still appears in Downloads when I want to use a different route.

When you download a torrent, it is stored in the path you specify, but an empty folder is still created in the Downloads folder in Windows 10.

Although I deleted that folder, it still appears when I open FrostWire and, although I touch the Apply button with the new route I want to use, this behavior does not change.

**1-** Change the path
**2-** Apply settings
**3-** Close FrostWire
**4-** Delete FrostWire downloads directory of: `Users\Name\Downloads`
**5-** Open FrostWire
**6-** Directory is create in Downloads again

![cap_001](https://user-images.githubusercontent.com/31228150/45198747-8b532a80-b23e-11e8-90d7-070d70802b3d.png)

![cap_002](https://user-images.githubusercontent.com/31228150/45198755-8f7f4800-b23e-11e8-91ef-3a88a6cb9be0.png)

_* I do not know if this is a repeated problem. I did a quick search of the problems that are open and those that have already been closed, with words like: "default download directory", and I did not see anything related. At least I did not find it._

**Comment by gubatron** [comment]
@aldenml please take a look at this

**Comment by gubatron** [comment]
@rpxomi thank you so much for this bug report

**Comment by muckachina** [comment]
I could reproduce this behavior but this seems to be a FrostWire Setting when you install the app since that folder is created "by Default" so you can always "_Restore Defaults_" folder within FrostWire Library or click the button "_Use Default_" at "Default Save Folder". 

You will find this address with the option "Use Default" even at _FrostWire Setup Wizard_

**Comment by rpxomi** [comment]
Ah, ok. It makes sense to be able to go back to a previous configuration.

**Why choose an option?**
If I want to use another folder, and "by default" the program will continue to create that folder for the purpose of restoring the configuration in the future.

**Why have an empty folder in the system?**
It would make sense for that folder to appear, once someone chooses to go back to the default configuration.

I uninstalled the program, checked if there were any folders with any configuration (Temp, Appdata, etc.), reinstalled FrostWire and now I chose the path from the wizard... and the program still creates it.

![wizard_001](https://user-images.githubusercontent.com/31228150/45240553-7d91b980-b2bf-11e8-952d-8d71ea994275.png)

![wizard_002](https://user-images.githubusercontent.com/31228150/45240555-7d91b980-b2bf-11e8-8738-d5ea7108af6a.png)

![wizard_003](https://user-images.githubusercontent.com/31228150/45240556-7d91b980-b2bf-11e8-8b40-19d4484ffc50.png)

It is a strange configuration to give an option and, at the same time, not to be able to choose it completely. I think that for my part, I will continue to delete the folder every time it appears until I get tired and I will ignore it.

**Comment by gubatron** [comment]
I agree, will attempt a fix soon. Thanks @muckachina for confirming replication
